modules/system/classes/UpdateManagerOld.php

```diff
@@ -0,0 +1,1100 @@
+<?php
+?>
```

Comment:
To be removed

Reply:
See: https://github.com/wintercms/winter/pull/1259/commits/f950ce3b8e735ed574fe23e97bae5e9b7bcbb2e7